### PR TITLE
use assent tests to check public api contract changes

### PR DESCRIPTION
**Pull request by colombod**
using assent as the package time tests doesn't capture nicely the nature of the change and is also too sensible to package signatures

### CODE REVIEW

File: dotnet-interactive.sln [hunk 1]
```diff
@@ -498,6 +500,18 @@ Global
...
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Debug|x64.Build.0 = Debug|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Debug|x86.Build.0 = Debug|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Release|Any CPU.Build.0 = Release|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Release|x64.ActiveCfg = Release|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Release|x64.Build.0 = Release|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Release|x86.ActiveCfg = Release|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Release|x86.Build.0 = Release|Any CPU
```
Comment (jonsequitur) [inline dotnet-interactive.sln:514]: Why add projects for this? I think it's a little more maintainable to embed these into the library-specific test suites.

Comment (colombod) [reply dotnet-interactive.sln:514]: i did add nothing there :( let me clean up

Comment (colombod) [reply dotnet-interactive.sln]: the only thing is that this project loads all, including kql, sql.

Comment (colombod) [reply dotnet-interactive.sln]: I'd prefer to have an api test on its own.

**Review by jonsequitur — APPROVED** [review]

**Pull request merged**